cfgov/legacy/views/__init__.py

```diff
@@ -23,11 +22,11 @@ def dbrouter_shortcut(request, content_type_id, object_id):
                 {'ct_id': content_type_id}
             )
 
-        if content_type.app_label in cfgov_apps:
-            obj = content_type.get_object_for_this_type(pk=object_id)
-        else:
+        if content_type.app_label in settings.LEGACY_APPS:
             obj = content_type.model_class().objects.db_manager(
                 'legacy').get(pk=object_id)
+        else:
+            obj = content_type.get_object_for_this_type(pk=object_id)
```

Comment:
I *think* we can actually drop this functionality. It addressed a corner case where Django Admin's "View on site" didn't play nicely with database routing (I *think* because it consults the django_content_type table)

We did that for (old) Ask CFPB, which no longer exists